### ISSUE TITLE
Survey, single-choice question: make "Option" text field longer

### ISSUE THREAD

**Issue by SebastianLopienski** [indico/indico#2968]
When adding a single-choice question (and maybe other types of questions) to an Indico survey, I can define several options for users to choose from. Unfortunately, the input text field for adding an option is quite short, so I don't see the whole phrase I'm typing in:

![image](https://user-images.githubusercontent.com/2960711/27175921-fc957502-51c0-11e7-9e11-b41e8da79aea.png)

Could this text field be longer please?

**Comment by SebastianLopienski** [comment]
Do you have any news for this issue?

**Comment by jouvin** [comment]
@SebastianLopienski I'd say the length limit was removed fixing https://github.com/indico/indico/issues/3119. It was for multiple choice but hopefully it applies also to single choice... Have you checked?

**Comment by SebastianLopienski** [comment]
@jouvin Just checked - CERN's Indico instance still has a short box. I can type a long text (so there is no limit of the number of characters, or this limit is big), but the box is small and I can only see part of it.
![screen shot 2018-04-06 at 10 50 37](https://user-images.githubusercontent.com/2960711/38412001-a6160d8a-3988-11e8-8c75-31bca705426f.png)

**Comment by jouvin** [comment]
@SebastianLopienski you are right, the fix I mentioned removed the limit on the number of character but didn't make the text box to enter it longer and the display of the text may also remain truncated in some context like yours (works for multiple choices)
